frontend/src/metabase/collections/components/CollectionHeader/CollectionUploadInfoModal.tsx

```diff
@@ -44,7 +41,7 @@ export const UploadInfoModal = ({
                 primary
                 role="link"
               >
-                {t`Enable in settings`}
+                {t`Go to setup`}
```

Comment:
The idea behind having `Enable in settings` was that it helps you know that it's not currently enabled. I think it's a faster read than "Go to setup" but curious to hear more about the change.